index.js

```diff
@@ -0,0 +1,35 @@
+#! /usr/bin/env ./node_modules/.bin/babel-node
+
+
+import nomnom from 'nomnom';
+import glob from 'glob';
+import path from 'path';
+
+
+// Find all available scaffolds
+const scaffolds = {};
+glob.sync('scaffold/*/', {realpath: true})
```

Comment:
OK. Need to add this rule to ESLint http://eslint.org/docs/rules/newline-after-var

And we need eslint here as well.